config/rules.toml

```diff
@@ -15,3 +15,14 @@
         'local1'
     ]
 
+[[rule]]
+    regexs = [
+        '^abc.*'
+    ]
+    prefixes = [
+        'xyz',
+        'poi'
+    ]
```

Comment:
I am not sure that we want both 'regexs' and 'prefixes' to be specified in the same rule. 
We introducing prefixes to speed-up the parsing process and in this regard combination of both does not make any sense. Except the case when prefix is not flexible enough to match what we want and then prefix can be used to filter out the majority of unmatched metrics, so the rest can be filtered by a regex. 
As I understand - this is not a case, so I suggest to make it explicit either regexs or prefixes, not both.